horovod/common/common.h

```diff
@@ -211,6 +214,8 @@ class OpContext {
                      std::shared_ptr<PersistentBuffer>* tensor) = 0;
   virtual Status AllocateOutput(TensorShape shape,
                                 std::shared_ptr<Tensor>* tensor) = 0;
+  virtual Status AllocateZeros(int64_t num_elements, DataType dtype,
```

Comment:
Can you add stub implementations of this method to the other `OpContext` children, including `TFOpContext` in `tensorflow/mpi_ops.cc` and `MXOpContext` in `mxnet/adapter.{h,cc}`?  These implementations should raise an error similar to the one for PyTorch <1.0.

Reply:
Done for TF and MXNet.